docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -6,6 +6,10 @@
 [id="mta-web-assessing-apps_{context}"]
 = Assessing an application by using the MTA assessment questionnaire
 
+The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other features. In the {ProductShortName} user interface, you can assess an application and display the currently saved assessments by using the Assessment module.
```

Comment:
Shall we at all mention that the assessment feature has been separated? This kind of info is more for Release Notes rather than this kind of product doc. Can we remove? I'd only leave some "process-related" info as an intro to the procedure. For example, we could repeat some "why do I need to perform the assessment" info from [Assessing and analyzing applications](https://deploy-preview-870--windup-documentation.netlify.app/docs/web-console-guide/master/#assessing-and-analyzing-applications) module. WDYT?

Reply:
I agree that "has been separated" sounds like a changelog and shouldn't be in the product docs

Reply:
Will be removed as per the discussion in the gdoc file.